sdk/network/armnetwork/go_mod_tidy_hack.go

```diff
@@ -0,0 +1,12 @@
+// +build modhack
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// This file, and the github.com/Azure/azure-sdk-for-go import, won't actually become part of
+// the resultant binary.
+
+package armcompute
```

Comment:
armnetwork

Reply:
Oops i forgot to change it when i copied it